spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  before(:each) { @bill = create(:bill) }
+
+  describe 'GET #index' do
+    before { get :index }
+
+    it { is_expected.to respond_with :ok }
+    it { is_expected.to render_template :index }
+    it { expect(@bill).to_not be nil }
+  end
+
+  describe "GET #new" do
+    before { get :new }
+
+    it { is_expected.to respond_with :ok }
+    it { is_expected.to render_template :new }
+  end
+
+  describe "POST #create" do
+
+    it "should create bill" do
+      user = create(:user)
+      sign_in user
+      bill = FactoryGirl.attributes_for(:bill).merge(user_id: user.id, units: {number_occupants: 4})
```

Comment:
Now that we've specified `include FactoryGirl::Syntax::Methods` in our RSpec config, we could just do `attributes_for` w/o the FG in front.